wiki/Ranking_Criteria/osu!catch/en.md

```diff
@@ -37,21 +38,20 @@ Overall rules and guidelines apply to every kind of osu!catch difficulty.
 
 #### Rules
 
-- **Your map must theoretically be possible to SS.** This means it must be possible to catch absolutely all fruits, including droplets.
-- **Edge dashes must not be used in direct conjunction with hyperjumps.** This is because such patterns require especially precise movement and force an unreasonable restriction on accuracy required to catch them.
-- **Each map must use at least two different combo colours which must not blend in with the map's background/storyboard/video.** This is so hit objects are always visible to the player.
+- **Your map must theoretically be possible to SS.** This means it must be possible to catch absolutely all fruits, drops and droplets.
+- **Each map must use at least two different combo colours which must not blend in with the map's background/storyboard/video.** This is so objects are always visible to the player.
 - **Do not use keysounds without hitnormal support.** If sounds blend perfectly with the song, feedback for hitting notes is minimal.
 
 #### Guidelines
 
 - **All circles and slider heads should represent a sound existing in the music.** This is usually a distinct sound, but can also represent a continuous sound with an indistinguishable start or end.
+- **Sliderends of extended sliders should be snapped according to the song's beat structure.** If the song is using a straight beat, 1/4, 1/8, and 1/16 should be used. If the song is using a swing beat, 1/6 or 1/12 should be used. If the song has a sound in a different snap from what was recommended, snapping to an actual beat always takes priority.
```

Comment:
are these called "sliders" and "circles" in catch? terminology seems mixed

if that is correct, I feel like "slider end" is preferred over "sliderend" but u can check with osu!standard rc for that idk

Reply:
yes they are

Reply:
There's no term in the osu!catch glossary that uniquely refers to circles, slider heads and sliderends on their own, so we went with the editor term. 

The osu!standard RC uses sliderend(s), but it also uses slider tails.